components/form/demo/advanced-search-form.md

```diff
@@ -114,7 +114,7 @@ ReactDOM.render(
   border-radius: 6px;
 }
 
-/* becasue the label width is not definite, you need to adjust it to center by yourself */
+/* because the label width is not fixed, you may need to adjust it */
```

Comment:
Hi Gray,
I just discovered this UI library and felt in love with it. So, I was in a hurry to start contributing.
You're right, I wasn't sure about the meaning of that statement.
Let me play a little bit with this example and get back to this case.